types/store/plugins.ts

```diff
@@ -76,6 +77,10 @@ export type Menu = {
 export type PluginComponent = {
     id: string;
     pluginId: string;
+
+    /** @default null - which means 'channels'*/
+    supportedProductIds?: ProductScope;
```

Comment:
The default param in the registry call for this is `null`, so if unspecified, it will follow the current behavior of Channels-only, but if you manually specify `undefined`, `inScope` will return false. Not sure if that level of granularity is useful, but given the current significance of `null`, I thought it was good to keep a "nothing" option.

Reply:
Makes sense. This definitely isn't the only area that we have `type | undefined | null` as a type. As long as we don't start needing to differentiate unset vs undefined vs null (since that's possible in objects and arrays), this is probably fine